libs/vm-modules/include/vm_modules/ml/model/model.hpp

```diff
@@ -143,6 +144,7 @@ class VMModel : public fetch::vm::Object
                                            math::SizeType const &                   inputs,
                                            math::SizeType const &                   hidden_nodes,
                                            fetch::vm::Ptr<fetch::vm::String> const &activation);
+  void LayerAddFlatten(fetch::vm::Ptr<fetch::vm::String> const &layer);
```

Comment:
this is fine for now, but we should take note that as soon as we add implement another layer that takes no inputs (except the string) we're going to have to refactor these layers to a slightly less clear format; something like 'LayerAddOneParam(Ptr<String> const &layer)' which has an if/else to determine whether to call Flatten, or another layer type. @vhavrylov-ncube previously implemented a version of this